src/components/JellyText/JellyText.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import uuid from 'react-uuid';
+import './jellyLetter.scss';
+
+function JellyText({ text }) {
+  return (
+    <>
+      { [ ...text ].map((letter) => (
```

Comment:
You don't need to make a clone of the array

```suggestion
      { text.map((letter) => (
```

Reply:
I'm not sure this would work(at least in this form), because text is a String, not an array. Making this change gives me the following error: `text.map is not a function`

My intention here was to convert the string into an array containing every letter from text (i.e. `"Title" -> [ "T", "i", "t", "l", "e" ]`).

Reply:
ah, then you should use the `split` function